docs/configuration_file.md

```diff
@@ -137,6 +137,9 @@ Some options are also mandatory.
 
 * `can_orchestrate` (*optional*, `boolean`) — allows using orchestrator build, default is false
 
+* `info_url_format` (*optional*, `string`) — format for `url` dockerfile label, by default :
+`https://access.redhat.com/containers/#/{authoritative-source-url}/{name}/images/{version}-{release}`, you can use any label,
```

Comment:
Maybe we should set this to a Docker Hub by default, being an opensource project? `https://hub.docker.com/{name}`